### PR TITLE
#[3]dryville additions

### PR DESCRIPTION
**Pull request by janetrbarclay**
Added 3 sections to the dryville story ("Getting Water to Your Homes", "Dryville's First Water Works", and "Be Gone, Dirty Water")

### CODE REVIEW

File: dryville_story.md [hunk 1]
```diff
@@ -15,3 +15,22 @@ So let's say you made the big move. You and some friends have found your (desola
...
 
 In your free time go ahead and start digging a hole in the ground for a [well](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-wells). If you dig deep enough you might hit the [water table](https://www.usgs.gov/special-topic/water-science-school/science/aquifers-and-groundwater), where there will be standing water. You'll be able to lower a bucket down to get the available [groundwater](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater). Things are starting out great -- you've already started using surface water (the creek and pond), ground water (your well), and you've even created your own water-storage system (the reservoir). So you now have a reliable source for your water needs. Time to relax? Not yet.
+
+##Getting Water to Your Homes
```
Comment (lindsayplatt) [inline dryville_story.md]: Add a space between the header format `##` and the header text. Without the space, the markdown formatting doesn't actually pick up that you are using the `##` syntax to create a header. You can check this by clicking the "Display the rich diff" button (see screenshot below), which will show you the formatted Markdown file. You don't have headers now, but that should change when you commit a space between `##` and the title text. 
 
image

File: dryville_story.md [hunk 2]
```diff
@@ -15,3 +15,22 @@ So let's say you made the big move. You and some friends have found your (desola
...
+
+But you still have the problem of keeping the storage tank full. Lugging water up to it is no better than lugging it up to your house; so you need to find a way to pump the water from the [creek](https://www.usgs.gov/special-topic/water-science-school/science/rivers-streams-and-creeks) uphill into the storage tank. Since Dryville doesn't have any industry to produce products to sell to the outside world to bring in money to buy things with yet, you need to build your own pump. And you can't use electricity since you haven't built a power plant yet (but you will). Can you think of a source of energy that can run your pump? A windmill could do the job, but in the desert there isn't always wind. Your creek! The water in your creek is flowing nicely -- so you build a paddle wheel in the river. The paddle spins in the creek's current and turns a rod that runs your homemade pump. Voila! You've got water being pumped uphill into your storage tank where gravity lets it flow to your homes.
+
+##Dryville's First Water Works
```
Comment (lindsayplatt) [inline dryville_story.md]: Include a space here, too!

File: dryville_story.md [hunk 3]
```diff
@@ -15,3 +15,22 @@ So let's say you made the big move. You and some friends have found your (desola
...
+
+Soon you start getting money from the citizens buying water from you - and that gives you an idea. You're going to write an acquaintance back home and tell him he needs to build another town down the road from Dryville. Then when your friend gets his town started, Dryville will offer to sell them water that you get from your public-supply system! You can build an [aqueduct system](https://www.usgs.gov/special-topic/water-science-school/science/aqueducts-move-water-past-and-today) to move water from Dryville to them. Of course, there will be a mark up to cover the cost of delivering the water — nothing wrong with making a little profit.
+
+##Be Gone, Dirty Water
```
Comment (lindsayplatt) [inline dryville_story.md]: I think you want to delete this section. You should only have two in this first PR. This third section will come later in #10

**Review by lindsayplatt — CHANGES_REQUESTED** [review]
Hi, Janet 👋 It's nice to meet you :)

**Review by lindsayplatt — APPROVED** [review]
Yes, looks good! 🎉

**Comment by janetrbarclay** [comment]
Thanks! And nice to meet you as well :)

**Pull request merged**